include/macros.h

```diff
@@ -32,6 +32,8 @@
 
 #define CURRENT_DAY (gSaveContext.day % 5)
 
+#define CLOCK_TIME(hr, min) ((s32)(((hr) * 60 + (min)) * 0x10000 / (24 * 60)))
```

Comment:
Isn't there already a macro for this, or am I thinking of a different macro?

Reply:
Not that I know of.
Also, I grabbed this macro from kyle and his invadepoh PR :sweat_smile:, maybe you saw it there